### PR TITLE
refactor: Remove core dependency

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**
Some code was brought back from core to prevent interlocking dependencies and responsability spreading.

This is a breaking change, core analog PR is here https://github.com/Infomaniak/ios-core/pull/92

### CODE REVIEW

**Review by adrien-coye — COMMENTED** [review]
I did a quick review, I have not finished, will tomorrow morning.

I agree with fighting `responsability spreading` in general, here the idea was to keep our `core-ios` with the least dependencies possible.

If there exists a need to decouple more auth and core, we can probably abstract some more if we have the time. I would argue this is orthogonal of having login a dependency of core.

File: Sources/InfomaniakLogin/DeleteAccountViewController.swift [hunk 1]
```diff
@@ -25,6 +25,8 @@ public protocol DeleteAccountDelegate: AnyObject {
 }
 
 public class DeleteAccountViewController: UIViewController {
+    @LazyInjectService var infomaniakLogin: InfomaniakLoginable
```
Comment (adrien-coye) [inline Sources/InfomaniakLogin/DeleteAccountViewController.swift:28]: Cool, no direct link to Core and we use DI. Nice.

**Review by adrien-coye — APPROVED** [review]
All good here 👍

File: Sources/InfomaniakLogin/InfomaniakLogin.swift [hunk 2]
```diff
@@ -16,14 +16,21 @@
...
-import InfomaniakCore
 import InfomaniakDI
 import SafariServices
 import WebKit
 #if canImport(UIKit)
 import UIKit
 #endif
 
+public enum Constants {
+    public static let DELETEACCOUNT_URL =
```
[the remark this answers is not in the document]
Comment (PhilippeWeidmann) [reply Sources/InfomaniakLogin/InfomaniakLogin.swift:27]: This is old code. It was shared with the Kotlin counterpart, I will rename it 👍

**Pull request merged**